### PR TITLE
Implement ./absolute clean command

### PR DESCRIPTION
**Pull request by sapzape**
I added a command named clean.
This command deletes all outputs in out/ directory.
I also added test about clean command.

ISSUE=#471

### CODE REVIEW

File: bootstrap/test/clean.test.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import {} from 'jest';
+import * as childProcess from 'child_process';
+import * as fs from 'fs';
+
+test('GET /test', async() => {
+    childProcess.spawnSync('./absolute', ['build_server']);
```
Comment (romandev) [inline bootstrap/test/clean.test.ts]: Could you try childProcess.spawnSync('./absolute', ['build_server'], { shell: true });?

File: bootstrap/test/clean.test.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+    expect(fs.existsSync('./out')).toBe(true);
+
+    childProcess.spawnSync('./absolute', ['clean']);
```
Comment (romandev) [inline bootstrap/test/clean.test.ts]: ditto.

**Comment by romandev** [comment]
@nadongguri Could you check why this is broken in Windows?
I guess the problem might be path problem coming from different file system.
The following similar test worked well in Absolute v1. Could you please check it with ```path```?
https://github.com/lunchclass/absolute/blob/absolute_v1/bootstrap/test/test_update.js#L46

(I tried but my Windows system is not working well now..)

**Comment by nadongguri** [comment]
@romandev, I will check it.

File: bootstrap/test/clean.test.ts [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import {} from 'jest';
+import * as childProcess from 'child_process';
+import * as fs from 'fs';
+
+test('GET /test', async() => {
+    childProcess.spawnSync('./absolute', ['build_server'], { shell: true });
```
Comment (nadongguri) [inline bootstrap/test/clean.test.ts]: that line doesn't work properly on Windows targets, so run ./absolute command as
childProcess.spawnSync(path.resolve('./absolute'), ['build_server'], { shell: true });

File: bootstrap/test/clean.test.ts [hunk 4]
```diff
@@ -0,0 +1,11 @@
+import {} from 'jest';
+import * as childProcess from 'child_process';
+import * as fs from 'fs';
+
```
Comment (nadongguri) [inline bootstrap/test/clean.test.ts]: Add import *as path from 'path';

File: bootstrap/test/clean.test.ts [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+test('GET /test', async() => {
+    childProcess.spawnSync('./absolute', ['build_server'], { shell: true });
+    expect(fs.existsSync('./out')).toBe(true);
+
+    childProcess.spawnSync('./absolute', ['clean'], { shell: true });
```
Comment (nadongguri) [inline bootstrap/test/clean.test.ts]: Add path.resolve the same as described above.

**Comment by sapzape** [comment]
When i modified code 'childProcess.spawnSync('absolute', ['build_server'], { shell: true });' then, it work.
I think if the OS is Windows, then it seems to use 'absolute' as a parameter.

**Pull request merged**